content/en/docs/instrumentation/js/exporters.md

```diff
@@ -67,6 +67,109 @@ $ docker run -d --name jaeger \
   jaegertracing/all-in-one:latest
 ```
 
+### Usage with the WebTracer
+
+When you use the OTLP exporter in a browser-based application, you need to note
+that:
+
+1. grpc-based exporting is not supported
```

Comment:
I believe http/proto is also not supported (today) either, so that means if you wish to send to some other OTLP endpoint (let's assume it doesn't require any authentication so api keys or whatever aren't a concern) - then you'll need to ensure that backend also supports http/json

Reply:
You are right, there's a third package for protobuf. Let me fix that.